### ISSUE TITLE
Is it possible to force Fabric to stop sending crash report temporary? 

### ISSUE THREAD

**Issue by hungdoan2** [sarriaroman/FabricPlugin#40]
I'm finding an option to disabled Fabric temporary - to make sure It doesn't send any thing to Fabric in development/testing mode.

And Fabric will be enabled again in distribution mode.

**Comment by Justin-Credible** [comment]
No, there is no way to temporarily disable the plugin. You can, however, use one of the two solutions below to differentiate your dev builds from your production builds.

Option 1: Set a key/value pair that indicates if it is a development build. These key/value pairs show up in the crash details for each crash entry.

    window.fabric.Crashlytics.setBoolValueForKey(true, "isDevelopmentBuild");

Option 2: Use a different package ID for your development builds. For example, `com.company.yourapp` for production builds and `com.company.yourapp.dev` for development builds. Fabric will treat the package ID as a different app and completely seperate all crashes and answers events in the Fabric console.

**Comment by hungdoan2** [comment]
@Justin-Credible Thanks you for your suggestions.
I think I will stick with "Option 2" as an temporary solution.

**About this feature**. I found this document for android: https://docs.fabric.io/android/crashlytics/build-tools.html
I wonder if we can implement this feature!.
According to the document, There are 2 options to disabled Fabric: 
+ At build time : We can implement a build hooks to update build configuration
+ At run time:  Or disabled fabric at run-time.

**Comment by losbeekos** [comment]
It would be nice if we could disable Fabric on dev builds though, we just don't want to clutter our crash reporting stats.
